android/app/src/googleplay/java/org/fossasia/openevent/fragments/MapsFragment.java

```diff
@@ -180,42 +176,41 @@ private void showLocationsOnMap() {
         });
     }
 
-    private void focucOnMarker(Marker marker){
-        marker.showInfoWindow();
```

Comment:
Why are you removing it? 
it reverts on of the feature of https://github.com/fossasia/open-event-android/pull/1749 which is Showing marker info on clicking on suggestion from search